submissions/Khatsur/html-css-popup/style.css

```diff
@@ -0,0 +1,116 @@
+body {
+  font-family: Arial, sans-serif;
+}
+
+nav {
```

Comment:
Avoid styling by HTML tags except if you add base styles. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)